lib/code_corps/github.ex

```diff
@@ -12,4 +12,33 @@ defmodule CodeCorps.Github do
     |> User.github_associate_changeset(params)
     |> Repo.update()
   end
+
+  def create_issue(attributes, project, current_user) do
+    access_token = current_user.github_access_token || default_user_token() # need to create the Github user for this token
+    client = Tentacat.Client.new(%{access_token: access_token})
+    response = Tentacat.Issues.create(
+      project.github_owner,
+      project.github_repo,
+      issue_attributes(attributes),
+      client
+    )
+    case response.status do
+      201 ->
+        response.body["id"] # return the github id
+      _ ->
+        # log error
+        nil
+    end
+  end
+
+  defp issue_attributes(attributes) do
+    %{
+      title: attributes[:title],
+      body: attributes[:body]
+    }
+  end
```

Comment:
Pulled it out because I was thinking of reusing it for the future `update_issue` function. Also was thinking we might expand it to be more fields (when we want to be able to assign users, labels, etc.)